scilpy/tractograms/streamline_and_mask_operations.py

```diff
@@ -36,10 +42,12 @@ def get_endpoints_density_map(streamlines, dimensions, point_to_select=1):
 
 def get_head_tail_density_maps(streamlines, dimensions, point_to_select=1):
     """
-    Compute two separate endpoints density maps for the head and tail
+    Compute two separate endpoints density maps for the head and tail of
+    a list of streamlines.
+
     Parameters
     ----------
-    streamlines: list of ndarray
+    streamlines: list of np.ndarray
         The list of streamlines to compute endpoints density from.
```

Comment:
They must be in voxel space, corner origin.

(I didn't check: would it be easy to send the sft instead, and do to_vox ourselves?)

Reply:
Very easy, will fix.